### PR TITLE
Add support for attribute examples

### PR DESCRIPTION
**Pull request by Kukunin**
Solves #27

### CODE REVIEW

**Comment by Bugagazavr** [comment]
Hello, thank you please add this PR to `CHANGELOG.md`

**Comment by Bugagazavr** [comment]
Do not worry about rubocop, i will fix this after this pull request will be merged

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 #### Features
 
-* Your contribution here.
```
Comment (Bugagazavr) [inline CHANGELOG.md]: Please return this string back

**Comment by Kukunin** [comment]
@Bugagazavr here we go

**Comment by Bugagazavr** [comment]
Good job, thank you

**Pull request merged**